src/AddressMetadata.php

```diff
@@ -13,11 +13,26 @@ public function __construct($name, $attribute = null, $resolveCallback = null)
         parent::__construct($name, $attribute, $resolveCallback);
 
         $this->withMeta([
+            'captureResponse' => false,
             'disabled' => false,
             'invisible' => false,
         ]);
     }
 
+    public function fromResponse(): self
+    {
+        return $this
+            ->withMeta(['captureResponse' => true])
+            ->resolveUsing(function () {
+                return json_encode(optional($this->resource)->address);
```

Comment:
You get the address field, but the property will not necessarily be called like that